Created FindSumOfCubesOfNumbersByRecursion.java

vedha-vikash-pixel opened the pull request:
Returns the sum of cubes of elements in the array

### Why:
Closes codinasion/codinasion#1404 

### What's being changed:
Wrote a program to get the sum  of cubes of elements in an array

vedha-vikash-pixel committed:
Created FindSumOfCubesOfNumbersByRecursion.java

Returns the sum of cubes of elements in the array

harshraj8843 committed:
Merge branch 'master' into patch-3

harshraj8843 reviewed (changes requested):
Hey @vedha-vikash-pixel 

Please move program file to `program/find-sum-of-cubes-of-numbers-by-recursion` folder

vedha-vikash-pixel committed:
Merge branch 'master' into patch-3

vedha-vikash-pixel committed:
Created FindSumOfCubesOfNumbersByRecursion.java

Wrote a program to get the sum of cubes of elements in an array using recursion.

vedha-vikash-pixel committed:
Delete FindSumOfCubesOfNumbersByRecursion.java

harshraj8843 committed:
Merge branch 'master' into patch-3

harshraj8843 reviewed (approved):
Thanks for contributing 👍🏻
